src/components/assessment/assessmentShape.ts

```diff
@@ -166,6 +166,7 @@ type ExternalLibrary = {
 
 export type Library = {
   chapter: number;
+  variant?: Variant;
```

Comment:
`variant` has been kept as an optional property to avoid having to change all usages of Library types